config/routes.rb

```diff
@@ -19,7 +19,7 @@
   resources :workflows do
     resources :extractors
     resources :extractors, param: :key do
-      resource :extracts
+      resource :extract, except: [:edit]
```

Comment:
`:edit` is the GET route, do you want to except `:update` here too?

Reply:
yes i absolutely do, thank you for catching that. i confuse new/create and edit/update sometimes